XamarinCommunityToolkitSample/Resx/AppResources.Designer.cs

```diff
@@ -9,48 +9,35 @@
 
 namespace Xamarin.CommunityToolkit.Sample.Resx {
```

Comment:
CC @jfversluis ... this file is regenerated again
We should find a way how to handle it